test/basic.html

```diff
@@ -70,13 +70,14 @@
       });
 
       suite('with aria-label attribute set', function() {
-        setup(function () {
+        setup(function() {
           f = fixture('with-aria-label');
         });
 
-        test('defined aria-label is not overwritten by initial heading', function() {
-          assert.strictEqual(f.getAttribute('aria-label'), 'the-aria-label');
-        });
+        test(
+            'defined aria-label is not overwritten by initial heading', function() {
+              assert.strictEqual(f.getAttribute('aria-label'), 'the-aria-label');
+            });
```

Comment:
this closing bracket is indented questionably imo but such is life

Reply:
I agree, but i can see the reasoning. (aligning it with the function block).